apps/app-trade-agents/src/navigation/Root/types.ts

```diff
@@ -49,3 +49,8 @@ export type DebetStackParamList = {
 export type ReportStackParamList = RefParamList & {
   ReportList: undefined;
 };
+
+export type ShipmentStackParamList = RefParamList & {
+  ShipmentList: undefined;
+  OrderView: { id: string; routeId?: string; readonly?: boolean };
+};
```

Comment:
### Code Review
- The code patch introduces a new type `ShipmentStackParamList` in the file `types.ts`.
- It seems to be properly formatted and follows the existing pattern of other type definitions.
- No bug risks or security vulnerabilities identified.


Quality Rate: 7/10